dcbr-api/api/models.py

```diff
@@ -30,11 +28,17 @@ class Registration(models.Model):
         max_length=50, choices=REG_STATUS_CHOICES, default=ACTIVE
     )
 
+    def default_expiry_date():
+        expiry_date = datetime.now() + timedelta(days=365)
```

Comment:
I am still not sure we need to store the expiry date, rather than calculating it at runtime.
However, rather than hard-coding the expiry interval it would be best to refer to the the existing global variable in [settings.py](https://github.com/bcgov/agri-dcbr/blob/master/dcbr-api/dcbr/settings.py#L274), which is driven by an environment variable. A usage example can be found [here](https://github.com/bcgov/agri-dcbr/blob/master/dcbr-api/email_service/tasks.py#L29-L38).

Reply:
this field is to solve the business req't that the administrators need to see the expiry date